content/manuals/compose/how-tos/lifecycle.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Using lifecycle hooks with Compose
+linkTitle: Use lifecycle hooks
+weight: 20
+desription: How to use lifecycle hooks with Docker Compose
+keywords: cli, compose, lifecycle, hooks reference
+aliases:
+- /compose/hooks/
+---
+
+## Services lifecycle hooks
+
+As Docker Compose run a service container, the actual command being ran is managed
+by the combination of `entrypoint` and `command` as documented [here](https://docs.docker.com/engine/containers/run/#default-entrypoint). Those ley you manage
+container setup and termination, catching SITERM signal to process a graceful shutdown. In some circumstances anyway it is easier to consider those a separate
+command to be introduced as lifecycle hooks, to be ran just after the container
+is started, or just before Compose stop it.
+
+Another benefit for lifecycle hooks, is that hook command can run with additional
+privileges, typically run as `root` or even `privileged` while the container is
+configured to have lower privileges to enhance security.
+
+### Post-Start hooks
+
```

Comment:
> These hooks don't wait for the container's main task (entrypoint) to finish.

entrypoint never finish, as this would mean container stopped. Here entrypoint and hook run concurently, with no guarantee on when things happen